sympy/ntheory/primetest.py

```diff
@@ -303,6 +302,7 @@ def _lucas_extrastrong_params(n):
             return (D, P, 1)
         elif j == 0 and D % n:
             return (0, 0, 0)
+    raise ValueError("Found no Lucas parameters")
```

Comment:
The main thing to do here is to add an explicit type annotations to the functions rather than adding code so that a type checker infers the types.